Backup: add link to CLI section (#32914)
home-assistant/home-assistant.github.io: source/_includes/common-tasks/backups.md

2. `ha backups restore slugname` - restores a specific backup
3. `ha backups new --name nameofbackup` - create a backup

> Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).

Author: Why did you remove drop `ha help`?

> Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).

Author: > Consider using a more formal tone for documentation.
> 
> ```diff
> - Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).
> + For additional information about command line usage, please refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line).
> ```
> 
> Committable suggestion
> > ‼️ **IMPORTANT**
> > Carefully review the code before committing. Ensure that it accurately replaces the highlighted code, contains no missing lines, and has no issues with indentation.

@coderabbitai Why did you remove ha help?

> Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).

Author: > @c0ffeeca7, thank you for pointing out the omission. The initial suggestion aimed to enhance the formality of the documentation, but it inadvertently excluded the `ha help` command, which is indeed useful for providing immediate command line assistance.
> 
> Here's a revised suggestion that includes the `ha help` command while maintaining a formal tone:
> 
> ```
> For additional information about command line usage, you may use the `ha help` command or refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line).
> ```
> 
> This version ensures that users have both immediate help through the CLI and detailed guidance via the documentation link.

@coderabbitai For instructional content, use the indicative mood. Do not use expressions of politeness such as 'may' or 'please'

> Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).

Author: > @c0ffeeca7, thank you for your guidance. Based on your feedback, here's the updated text that adopts a more direct and authoritative tone:
> 
> ```
> Use the `ha help` command or refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line) for additional information about command line usage.
> ```
> 
> This adjustment ensures the instructional content is clear and direct, as per your specifications.

@coderabbitai The goal of an instruction should be fronted.

> Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).

Author: @coderabbitai Perfect! Can you remember these rules for future reviews?

> Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).

Author: > For instructional content in documentation, use a direct and authoritative tone. Avoid expressions of politeness such as 'may' or 'please', and ensure the goal of the instruction is fronted.

Can you apply the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/welcome/)?

> Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).

Author: > @c0ffeeca7, based on your feedback, here's the rephrased instruction with the goal fronted:
> 
> ```
> For additional information about command line usage, use the `ha help` command or refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line).
> ```
> 
> This version places the primary goal at the beginning of the sentence, aligning with your directive for clear and direct instructional content.

@coderabbit Provide your sentence as a suggestion

> Use `ha help` to get more information about the command line usage or refer to the documentation on the [Home Assistant Command Line](/common-tasks/os/#home-assistant-via-the-command-line).

Author: ```suggestion
For additional information about command line usage, use the `ha help` command or refer to the [Home Assistant Command Line documentation](/common-tasks/os/#home-assistant-via-the-command-line).
```
